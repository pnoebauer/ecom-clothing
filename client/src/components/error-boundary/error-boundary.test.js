import React from 'react';
import {shallow} from 'enzyme';

import ErrorBoundary from './error-boundary.component';

describe('ErrorBoundary component tests', () => {
	const wrapper = shallow(
		<ErrorBoundary>
			<div>Output if there are no errors</div>
		</ErrorBoundary>
	);

	wrapper.setState({hasErrored: true});

	test('render ErrorBoundary component', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
