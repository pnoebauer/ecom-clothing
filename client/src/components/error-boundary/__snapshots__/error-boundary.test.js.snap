// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary component tests render ErrorBoundary component 1`] = `
<error-boundarystyles__ErrorImageOverlay>
  <error-boundarystyles__ErrorImageContainer
    imageUrl="https://i.imgur.com/yW2W9SC.png"
  />
  <error-boundarystyles__ErrorImageText>
    Sorry this page is broken
  </error-boundarystyles__ErrorImageText>
</error-boundarystyles__ErrorImageOverlay>
`;
